src/components/organisms/EntityDetails/EntityDetailsLayer.tsx

```diff
@@ -50,24 +50,24 @@ const EntityDetailsLayer: FC<PropsWithChildren<EntityDetailsLayerProps>> = ({
   const [metrics, setMetrics] = useEntityDetailsField('metrics');
   const [, setCurrentPage] = useEntityDetailsField('currentPage');
   const [executions, setExecutions] = useEntityDetailsField('executions');
+  const [storeDetails] = useEntityDetailsField('details');
   const [, setIsFirstTimeLoading] = useEntityDetailsField('isFirstTimeLoading');
   const [daysFilterValue, setDaysFilterValue] = useEntityDetailsField('daysFilterValue');
 
   const isClusterAvailable = useSystemAccess(SystemAccess.agent);
-  const isSystemAvailable = useSystemAccess(SystemAccess.system);
   const wsRoot = useWsEndpoint();
 
   const {data: rawExecutions, refetch} = useGetExecutions(
     {id, last: daysFilterValue},
-    {pollingInterval: PollingIntervals.long, skip: !isSystemAvailable}
+    {pollingInterval: PollingIntervals.long, skip: !isClusterAvailable && Boolean(executions)}
```

Comment:
Such rule will:
* disable offline results, as won't fetch even initial data when agent is offline (`!isClusterAvailable`)
* until there will be no data it will be polling all the time, even if cluster is not available (`&& Boolean(executions)`

It won't work this way